README.Rmd

```diff
@@ -0,0 +1,93 @@
+```{python tags=c("hide-cell")}
+# this keeps the pandas dataframe repr from spitting out scoped style tags
+# which don't render on github
+import pandas as pd
+pd.set_option("display.notebook_repr_html", False)
+```
+
+# pins-python
+
+[![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/machow/pins-python/HEAD)
+
+The pins package publishes data, models, and other python objects, making it
+easy to share them across projects and with your colleagues. You can pin
+objects to a variety of pin *boards*, including folders (to share on a
+networked drive or with services like DropBox), RStudio Connect, and Amazon
+S3.
+Pins can be automatically versioned, making it straightforward to track changes,
+re-run analyses on historical data, and undo mistakes.
+
+## Installation
+
+```shell
+python -m pip install pins
+```
+
+## Usage
+
+See the [documentation](https://rstudio.github.io/pins-python) for getting started.
+
+To use the pins package, you must first create a pin board. A good place
+to start is `board_folder()`, which stores pins in a directory you
+specify. Here I’ll use a special version of `board_folder()` called
+`board_temp()` which creates a temporary board that’s automatically
+deleted when your R session ends. This is great for examples, but
```

Comment:
```suggestion
deleted when your Python session ends. This is great for examples, but
```
Is this still the right way to say this for Python?

Reply:
Ah--thanks for flagging. I'm not sure what language would feel natural to people--maybe "python script or notebook session ends"?